schema-registry/common/src/main/java/com/hortonworks/registries/schemaregistry/state/SchemaVersionLifecycleStates.java

```diff
@@ -15,6 +15,7 @@
  */
 package com.hortonworks.registries.schemaregistry.state;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
nit: unused imports, pl do optimize imports for this class.

Reply:
Done